src/components/Button.js

```diff
@@ -1,15 +1,30 @@
 import styled from 'styled-components';
 
 const Button = styled.button`
-  background-color: var(--red);
   border: none;
   border-radius: 6px;
+`;
+
+export default Button;
+
+const RedButton = styled(Button)`
+  background-color: var(--red);
```

Comment:
The problem with this naming convention of color variables is that you will have a lot of trouble if you ever want to change colors: Let's say you switch from black and red to a white / green theme (I know, you will not, but let's suppose so), then you have to rename your variables in the `:root{}` and in ALL files as well.
This is why it's better to use general names, e.g. `--bg-main`, `--bg-secondary`, `--button-text` etc.